apps/cow-fi/app/api/revalidate/route.ts

```diff
@@ -0,0 +1,34 @@
+import { revalidateTag } from 'next/cache'
+import { NextRequest, NextResponse } from 'next/server'
+
+// Secret key for protecting the revalidation endpoint
+const REVALIDATE_SECRET = process.env.REVALIDATE_SECRET
+
+export async function GET(request: NextRequest) {
```

Comment:
How is this used? Is it interesting to document and make a process for it?

Reply:
It's documented on Notion under title: `Edit content in CMS: Cache Invalidation Guide`